### DISCUSSION TITLE
Prim Duplication across Layers

### DISCUSSION THREAD

**Discussion by brycegbrazen** [Autodesk/maya-usd#1964] [Q&A] [answered]
Hi all,

When bringing in a set assembly as a reference to one of our shots, it appears we are no longer able to duplicate any of the prims defined on the reference when calling the standard Maya action to duplicate (Ctrl+D). I get the following error:

`Duplication of '<prim>' failed: Cannot duplicate [<prim>]. It is defined on another layer. Please set [<reference_set>],[<prim_layer1>],[<prim_layer2>] as the target layer to proceed.. // `

Can anyone explain why this error appears when trying to duplicate a prim across layers?
Also, is there any way to work around this so that I can duplicate these prims that are defined with nested composition arcs?

**Comment by kxl-adsk** [comment, marked as answer]
The basic duplicate (and other similar actions) implementation is only allowed to perform an action if it all happens within the scope of the current edit target. This gives the studio the ability to tightly control where the edits will go. Getting to a more advanced duplicate (via potentially different shortcut) is something we discussed in the past but haven't got to yet.
